htdocs/core/triggers/interface_50_modTicket_TicketEmail.class.php

```diff
@@ -182,7 +182,7 @@ public function runTrigger($action, $object, User $user, Translate $langs, Conf
 
 				$langs->load('ticket');
 
-				$object->fetch('', $object->track_id); // Should be useless
+				$object->fetch('', '', $object->track_id); // Should be useless
```

Comment:
The $object should already be the object ticket received by the trigger. Did you try by removing this fetch() ?  Comment says it is useless.

Reply:
You're right.
It works fine without it and object was not found (track id was'nt in the third parameter).